extensions/wikia/ContentReview/ContentReview.hooks.php

```diff
@@ -240,7 +241,7 @@ public function onShowDiff( \DifferenceEngine $diff, &$notice ) {
 	public function onUserRightsGroupCheckboxes( $group, &$disabled, &$irreversible ) {
 		global $wgUser;
 
-		if ( $group === 'content-reviewer' && ( !$wgUser->isAllowed( 'content-review' ) || !$wgUser->isStaff() ) ) {
+		if ( $group === 'content-reviewer' && !$wgUser->isAllowed( 'content-review' ) ) {
```

Comment:
I think we wanted to check both `content-review` and staff in case we add some content reviewers who are not staff in the future, as they shouldn't be able to add the group.